core/queryplain.ml

```diff
@@ -0,0 +1,706 @@
+open Utility
```

Comment:
I'm not excited about the Query/Queryplain/Queryshredding nomenclature.  I gather that Query now has common code that both Queryplain and Queryshredding use, and that the latter differ in the "evaluation" (IR -> normalized QL) code?

I admit guilt for introducing the bad module name Queryshredding.  I suggest the following to expiate my sins:

- Query
- EvalQuery
- EvalNestedQuery

I also wonder if it might make sense to move query*.ml* into a subdirectory "query" or "database", as well.

Reply:
Renamed and moved to query/ subdirectory.